.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -949,10 +1042,16 @@ stages:
         command: "BuildLinuxIntegrationTests --framework $(publishTargetFramework)"
 
     - task: DownloadPipelineArtifact@2
-      displayName: Download linux tracer home
+      displayName: Download linux monitoring home for serverless
       inputs:
-        artifact: linux-tracer-home-$(baseImage)
-        path: $(System.DefaultWorkingDirectory)/tracer/serverlessArtifacts
+        artifact: linux-monitoring-home-$(baseImage)
+        path: $(System.DefaultWorkingDirectory)/tracer/shared/bin/monitoring-home
+
+    - task: DownloadPipelineArtifact@2
+      displayName: Download linux monitoring home
+      inputs:
+        artifact: linux-monitoring-home-$(baseImage)
+        path: $(System.DefaultWorkingDirectory)/shared/bin/monitoring-home
```

Comment:
I would like to download the monitoring only once, but I do not know how to let docker-compose/docker know where the artifact is :/